spec/v2/providers/fixtures.ts

```diff
@@ -1,4 +1,28 @@
+import { ManifestEndpoint } from "../../../src/runtime/manifest";
+import { TriggerAnnotation } from "../../../src/v2/core";
 import * as options from "../../../src/v2/options";
+import { RESET_VALUE } from "../../../src/common/options";
+
+export const MINIMAL_V2_ENDPOINT: ManifestEndpoint = {
+  availableMemoryMb: RESET_VALUE,
+  concurrency: RESET_VALUE,
+  ingressSettings: RESET_VALUE,
+  maxInstances: RESET_VALUE,
+  minInstances: RESET_VALUE,
+  serviceAccountEmail: RESET_VALUE,
+  timeoutSeconds: RESET_VALUE,
+  vpc: RESET_VALUE,
+};
+
+export const MINIMAL_V1_ENDPOINT: ManifestEndpoint = {
+  availableMemoryMb: RESET_VALUE,
+  ingressSettings: RESET_VALUE,
+  maxInstances: RESET_VALUE,
+  minInstances: RESET_VALUE,
+  serviceAccountEmail: RESET_VALUE,
+  timeoutSeconds: RESET_VALUE,
+  vpc: RESET_VALUE,
+};
```

Comment:
Unneeded? These are defined in https://github.com/firebase/firebase-functions/blob/master/spec/fixtures.ts#L25 instead.

Reply:
Oof, didn't see they were moved during conflict resolution.

Are you ok with:

```
export { MINIMAL_V1_ENDPOINT, MINIMAL_V2_ENDPOINT } from "../../fixtures";
```